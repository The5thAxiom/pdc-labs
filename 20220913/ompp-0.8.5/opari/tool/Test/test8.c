int main() {
  int i;

#pragma omp inst init

#pragma omp inst off

#pragma omp inst begin(foo)

#pragma omp parallel
      i = 9;

#pragma omp inst end(foo)

#pragma omp inst on

#pragma omp inst finalize

}
