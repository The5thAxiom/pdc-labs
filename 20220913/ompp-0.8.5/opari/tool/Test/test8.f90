      program test8

C$OMP INST INIT

      !$OMP INST OFF

      !$OMP INST BEGIN(foo)

      !$OMP PARALLEL
      i = 9
      !$OMP end PARALLEL

      !$OMP INST END(foo)

      !$OMP INST ON

   !$OMP INSTFINALIZE

      end
