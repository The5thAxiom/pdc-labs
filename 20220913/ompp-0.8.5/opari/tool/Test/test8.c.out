#ifdef _POMP
#  undef _POMP
#endif
#define _POMP 200110

#include "test8.c.opari.inc"
#line 1 "test8.c"
int main() {
  int i;

POMP_Init();
#line 5 "test8.c"

POMP_Off();
#line 7 "test8.c"

POMP_Begin(&omp_rd_22);
#line 9 "test8.c"

POMP_Parallel_fork(&omp_rd_23);
#line 10 "test8.c"
#pragma omp parallel POMP_DLIST_00023
{ POMP_Parallel_begin(&omp_rd_23);
#line 11 "test8.c"
      i = 9;
POMP_Barrier_enter(&omp_rd_23);
#pragma omp barrier
POMP_Barrier_exit(&omp_rd_23);
POMP_Parallel_end(&omp_rd_23); }
POMP_Parallel_join(&omp_rd_23);
#line 12 "test8.c"

POMP_End(&omp_rd_22);
#line 14 "test8.c"

POMP_On();
#line 16 "test8.c"

POMP_Finalize();
#line 18 "test8.c"

}
